receiver/splunkenterprisereceiver/README.md

```diff
@@ -8,7 +8,7 @@ jobs.
 
 ## Configuration
 
-The following settings are required, omitting them will either cause your receiver to fail to compile or result in 4/5xx return codes during scraping.
+The following settings are required, omitting them will either cause your receiver to fail to compile or result in 4/5xx return codes during scraping. These must be set for each Splunk instance type (indexer, search head, or cluster master) from which you wish to pull metrics. At present, only one of each type is accepted, per configured receiver instance. This means, for example, that if you have three different "indexer" type instances that you would like to pull metrics from you will need to configure three different `splunkenterprise` receivers for each indexer node you wish to monitor.
```

Comment:
Because I'm a docs dork... Mind putting this in its own paragraph? and/or following a `**NOTE:**` to REALLY DRIVE HOME this specific thing?

Reply:
yea makes sense to me!